### PR TITLE
Example project

### PR DESCRIPTION
**Pull request by emergent-joed**

### CODE REVIEW

**Review by kariheitz — COMMENTED** [review]
remove obj folder from repo.

File: Example/.gitignore [hunk 1]
```diff
@@ -0,0 +1,400 @@
+﻿## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+##
+## Get latest from https://github.com/github/gitignore/blob/main/VisualStudio.gitignore
```
Comment (kariheitz) [inline Example/.gitignore:4]: line four comment

**Review by emergent-joed — COMMENTED** [review]
Starting now

File: Example/Example.csproj [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```
Comment (milkDan) [inline Example/Example.csproj:4]: Please remove this line

**Review by kariheitz — COMMENTED** [review]
global review notes

File: Example/.gitignore (hunk 1 above)
Comment (emergent-joed) [reply Example/.gitignore:4]: No it looks good

File: Example/Example.csproj (hunk 2 above)
Comment (milkDan) [inline Example/Example.csproj:4]: Please remove line 4

**Review by milkDan — APPROVED** [review]
This looks great!

**Review by kariheitz — APPROVED** [review]
kari approves

**Pull request merged**